pkg/generate/app/cmd/newapp.go

```diff
@@ -218,6 +218,11 @@ func (c *AppConfig) SetOpenShiftClient(osclient client.Interface, originNamespac
 		ClientMapper: c.clientMapper,
 		Namespace:    originNamespace,
 	}
+	c.dockerSearcher = app.ImageImportSearcher{
```

Comment:
The setters is a questionable pattern, we probably need to make these
part of an initializer